testing/load-testing/get.js

```diff
@@ -0,0 +1,47 @@
+import { check } from "k6";
+import http from "k6/http";
+
+// This params will be used to make the GET request
+var params;
+
+// This headers and body will be used only for logging in the first time
+let log_in_headers = {
+  // E.g.
+  "Content-Type": "application/json",
+  "Accept": "application/json",
+  "Other": "header"
+}
+
+let login_body = {
+  // E.g.
+  sessions: {
+    username: "user",
+    password: "very_secret_password"
+  }
+};
+
+export default function() {
+
+  // This will be run only the first time, when the headers are not initialized
+  if(params == null) {
```

Comment:
Unitialized variables are not `null`, but `undefined`, so this shouldn't be working as you want it to.